windows-installer/Package.wxs

```diff
@@ -48,24 +66,25 @@
 			Return="check"
 			/>
 
-    <!-- Execution of uninstall command-->
-    <SetProperty
-      Id="RunUninstallAction"
-			Value='"[%SystemFolder]cmd.exe" /c "[%IDE_ROOT]\_ide\installation\bin\ideasy.exe -f uninstall > [%IDE_ROOT]\uninstall.log"'
-      Before="RunUninstallAction"
-      Sequence="execute"
-      />
-    <CustomAction
-      Id="RunUninstallAction"
-      BinaryRef="Wix4UtilCA_X64"
-      DllEntry="WixQuietExec"
-      Execute="deferred"
-      Impersonate="yes"
-      Return="ignore"
-      />
+		<!-- Execution of uninstall command-->
+		<SetProperty
+		  Id="RunUninstallAction"
+		  Value='"[%SystemFolder]cmd.exe" /c "[IDE_ROOT]_ide\installation\bin\ideasy.exe -f uninstall > [IDE_ROOT]uninstall.log"'
```

Comment:
Why did you have to change all this at all? This was already working before...
Removing the backslash looks like a bad idea. A duplicated backslash in a path will not hurt but a missing backslash will cause a severe issue. Also we do not `cd` to a defined location. You now write the log to the current working directory what is unspecified and may change with Windows versions or versions of Wix.